include/alpaka/alpaka.hpp

```diff
@@ -205,3 +205,8 @@
 // vec
 #include <alpaka/vec/Vec.hpp>
 #include <alpaka/vec/Traits.hpp>
+
+namespace alpaka {
```

Comment:
This looks outdated since you removed the namespaces.

Reply:
No, this is the current state of the PR. This PR has 2 using declarations in alpaka.hpp.